tsconfig.base.json

```diff
@@ -15,14 +15,17 @@
     "skipDefaultLibCheck": true,
     "baseUrl": ".",
     "paths": {
-      "@elewa-website/elements/layout/header": [
-        "libs/elements/layout/header/src/index.ts"
-      ],
       "@elewa-website/cards": ["libs/cards/src/index.ts"],
-      "@elewa-website/elements/cards": ["libs/elements/cards/src/index.ts"],
```

Comment:
This removes some of the aliased paths to a few modules in the codebase. Kindly don't remove them.

Reply:
My bad. This was Okay before. Just make sure there's only one aliased path set for the header module.